WordPress/Jetpack/Resources/release_notes.txt

```diff
@@ -1,6 +1,7 @@
-* [*] Login Flow: Provide ability for user to cancel login WP.com flow when already logged in to a self-hosted site [#19349]
-* [*] Weekly roundup: Adds support for weekly roundup notifications to the Jetpack app. [#19364]
-* [*] Fixed an issue where the push notifications prompt button would overlap on iPad. [#19304]
-* [*] Story Post: Fixed an issue where deleting one image in a story draft would cause the following image not to load. [#16966]
-* [*] Fixed an issue where the no result label on the side menu is oversize on iPad. [#19305]
-* [*] [internal] Various low level database refactors around posts, pages, and comments. [#19353, #19363, #19386]
+You can now cancel a WordPress.com login while you’re already logged in to a self-hosted site—no more getting stuck on the login screen if you change your mind.
```

Comment:
Praise (❤️): I noticed you changed `WP.com` with `WordPress.com` everywhere. I read your reasoning about that change and I agree, good call! 🥇